libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -239,28 +240,6 @@ static map<string, uint8_t> const builtins = {
 	{"i64.extend_i32_u", 0xad},
 };
 
-bytes lebEncode(uint64_t _n)
```

Comment:
Was this one also incorrect?

Reply:
This one is correct, I will replace this implementation with the original code.